test/Microsoft.ML.Tests/Transformers/CategoricalTests.cs

```diff
@@ -76,10 +76,10 @@ public void CategoricalOneHotHashEncoding()
             var mlContext = new MLContext();
             var dataView = mlContext.Data.LoadFromEnumerable(data);
 
-            var pipe = mlContext.Transforms.Categorical.OneHotHashEncoding("CatA", "A", 3, 0, OneHotEncodingEstimator.OutputKind.Bag)
-                .Append(mlContext.Transforms.Categorical.OneHotHashEncoding("CatB", "A", 2, 0, OneHotEncodingEstimator.OutputKind.Key))
-                .Append(mlContext.Transforms.Categorical.OneHotHashEncoding("CatC", "A", 3, 0, OneHotEncodingEstimator.OutputKind.Indicator))
-                .Append(mlContext.Transforms.Categorical.OneHotHashEncoding("CatD", "A", 2, 0, OneHotEncodingEstimator.OutputKind.Binary));
+            var pipe = mlContext.Transforms.Categorical.OneHotHashEncoding("CatA", "A", OneHotEncodingEstimator.OutputKind.Bag, 3, 0)
```

Comment:
>OutputKind [](start = 112, length = 10)

nit: I don't feel OutputKind is a good name.. Maybe `Representation`?

Reply:
I would prefer not to change it in this PR. Could you post an issue about it if you want to change the name?

---
In reply to: [266133597](https://github.com/dotnet/machinelearning/pull/2959#discussion_r266133597) [](ancestors = 266133597)

Reply:
Why not. [https://github.com/dotnet/machinelearning/issues/3005](https://github.com/dotnet/machinelearning/issues/3005)

---
In reply to: [266681667](https://github.com/dotnet/machinelearning/pull/2959#discussion_r266681667) [](ancestors = 266681667,266133597)